applications/tari_base_node/windows/runtime/start_tari_basenode.bat

```diff
@@ -22,7 +22,17 @@
 @echo Start Tor Services
 @echo ----------------------------
 @call %my_exe_path%\start_tor.bat
-@if [%errorlevel%]==[10101] goto :END
+@if [%errorlevel%]==[10101] (
+    @echo.
+    @echo It seems Tor could not be started properly.
+    @echo If '%my_exe%' still reports an error:
+    @echo   - Try to start Tor manually:
+    @echo     - execute 'start_tor.bat' from '%my_exe_path%', or
+    @echo     - select 'Tor Services' from the 'Tari - Testnet' menu
+    @echo   - Wait for '[notice] Bootstrapped 100% (done): Done' in the Tor console
+    @echo.
+    @pause
```

Comment:
This pause is probably also not necessary

Reply:
Tor not starting, or not starting fast enough, has been a major issue, so this will definitely help. The same reasoning as for the above comments applies, but in addition, this pause is strategical. The user will have the ability to look at the Tor console to see what is happening there, and if necessary, wait for Tor to complete its circuit and be 100% bootstrapped.